.github/workflows/validate_python.yml

```diff
@@ -29,8 +29,15 @@ jobs:
       with:
         poetry-version: 1.1.5
 
-    - name: Validate RFmx, XNET, and session examples
+    - name: Validate examples (except DAQmx)
       run: |
-        python source/codegen/validate_examples.py -p "*rfmx*"
+        python source/codegen/validate_examples.py -p "*dcpower*"
+        python source/codegen/validate_examples.py -p "*digital*"
+        python source/codegen/validate_examples.py -p "*dmm*"
+        python source/codegen/validate_examples.py -p "*fgen*"
+        python source/codegen/validate_examples.py -p "*rf*"
+        python source/codegen/validate_examples.py -p "*scope*"
+        python source/codegen/validate_examples.py -p "*switch*"
+        python source/codegen/validate_examples.py -p "*tclk*"
         python source/codegen/validate_examples.py -p "*xnet*"
         python source/codegen/validate_examples.py -p "session"
```

Comment:
Is there a way to do this in a single line? Maybe having a flag to exclude certain things and include everything else?

Reply:
Yeah, this was ugly. Updated to add a default for the `-p` arg to `"*"` so it includes everything by default. Added additional `-e` argument for exclusions so now this is one line `python source/codegen/validate_examples.py -e "nidaqmx"` to run everything except that one.